package.json

```diff
@@ -259,6 +260,7 @@
     "concurrently": "^6.0.0",
     "coveralls": "^3.1.0",
     "cross-env": "^7.0.3",
+    "crypto": "^1.0.1",
```

Comment:
Doesn't node have a built-in crypto module?

Reply:
haha there even was a deprecation notice 🙈 good catch